word_language_model/main.py

```diff
@@ -53,6 +64,12 @@
     if not args.cuda:
         print("WARNING: You have a CUDA device, so you should probably run with --cuda")
 
+if args.model == 'Transformer':
+    try:
+        from torch.nn import Transformer
+    except:
+        raise ImportError('Transformer module exists in PyTorch 1.1 and above.')
```

Comment:
I think it might be better to say "Transformer module does not exist in PyTorch 1.1 or lower".

Reply:
Move this into model.py